### ISSUE TITLE
[BUG] No containers to restart

### ISSUE THREAD

**Issue by ThaDaVos** [jesseduffield/lazydocker#258]
**Describe the bug**

When I try to restart one of my services in my docker-compose file I get the following screen:
![image](https://user-images.githubusercontent.com/5251028/119802740-19840900-bedf-11eb-9b75-4ce08a3b19d5.png)

I started lazydocker as follow `lazydocker -f .docker/docker-compose.yaml -f .docker/docker-compose.override.yaml`
We're using a specific project layout with our projects and a nice helper function for handling our docker stuff, this bash-function eventually calls lazydocker with the above layout but for some reason restarting of the services doesn't work...

**To Reproduce**
1. Start lazydocker with specific docker files
2. try to restart a service using the menu and choosing restart (also tried the restart options)

**Expected behaviour**
My container will be restarted

**Screenshots**
![image](https://user-images.githubusercontent.com/5251028/119803167-7b447300-bedf-11eb-9909-9285dc5b76d6.png)

**Desktop (please complete the following information):**
 - OS: Linux Ubuntu 18.04 running in WSL 2
 - Lazydocker Version [e.g. v0.1.45]: 0.12
 - The last commit id if you built project from sources (run : ```git rev-parse HEAD```)

**Additional context**

**Comment by denisfalqueto** [comment]
There's a workaround: you can edit lazydocker's config and add the following option:

    commandTemplate:
      dockerCompose: docker compose

This is due to a change in recent docker-compose distribution that made it a plugin for docker main command, instead of an independent command. Probably Lazydocker should try to handle that situation automatically?

**Comment by jesseduffield** [comment]
Yeah this indeed is something we should handle automatically. I wonder if `docker compose` has been around long enough that we can just expect people to have it? If not we'll need to do a check at startup to see whether we need to use the old approach
